app/tests/api/routes/test_LearnRequest.py

```diff
@@ -0,0 +1,68 @@
+from fastapi.testclient import TestClient
+
+
+def test_learn_request_under_limit(client: TestClient):
+    response = client.post(
+        "/learn",
+        json={
+            "text": "dog",
+        },
+    )
+    print(response.json())
+    assert response.status_code == 422
+
+    assert response.json() == {
+        "detail": [
+            {
+                "type": "string_too_short",
+                "loc": ["body", "text"],
+                "msg": "String should have at least 4 characters",
+                "input": "dog",
+                "ctx": {"min_length": 4},
+            }
+        ]
+    }
+
+
+def test_learn_request_right_on_inclusive_limit(client: TestClient):
+    response = client.post(
+        "/learn",
+        json={
+            "text": "cats",
+        },
+    )
+    assert response.status_code == 204
+
+    # assert response.json() == {"detail": [
```

Comment:
Let's remove debugging code such as this comment section and the print() at line 11 👍

Reply:
removed👍